src/__tests__/__snapshots__/index.js.snap

```diff
@@ -116,7 +122,7 @@ exports[`merges composed component styles for reasonable overrides 1`] = `
 }
 
 <div
-  class="other classes are not removed css-1k8eh41"
+  class="other classes are not removed css-1k8eh41 glamorous_glamorous_glamorous_div__"
```

Comment:
When you wrap a glamorous component multiple times like this you're `displayName` will be: `glamorous(glamorous(glamorous.div))` which isn't super easy to read. Unfortunately, because you cannot have `(` and `)` in class names, so this is even harder to read. However, many folks will hopefully be using the babel plugin so it should be more helpful :)